cmd/controller/main.go

```diff
@@ -168,6 +164,17 @@ func createReservationManager(client *k8s.KubeClient, log *logrus.Logger) (mgr c
 	if err = reservationController.SetupWithManager(mgr); err != nil {
 		return nil, err
 	}
+	wrappedK8SClient := k8s.NewKubeClient(client, log, *namespace)
 
+	kubeCache, err := k8s.InitKubeCache(log, ch,
```

Comment:
Can we use cached client?

Reply:
We create cache `kubeCache` here and use it further in `cachedCrHelper` in `Controller` for drive and lvg

Reply:
It's ok to use cache for AC, but not for LVG, so for LVG we use crHelper in capacity controller and for AC we use cacheCrHelper